cmd/feedback/feedback.go

```diff
@@ -80,17 +82,49 @@ type SurveyConfigInterface interface {
 // SurveyStore stores all available surveys.
 // New surveys should be added here.
 var SurveyStore = map[string]SlackSurvey{
-	// PlatformSurvey asks for general developer experience feedback
-	PlatformSurvey: {
-		Name:              PlatformSurvey,
-		PromptDisplayText: "Help make the Slack platform better",
-		PromptDescription: "Tell us about your experience as a Slack developer",
+	// SlackCLIFeedback asks for Slack CLI feedback using GitHub Issues
+	SlackCLIFeedback: {
+		Name:              SlackCLIFeedback,
+		PromptDisplayText: "Slack CLI",
+		PromptDescription: "Questions, issues, and feature requests about the Slack CLI",
+		SkipQueryParams:   true,
+		URL: url.URL{
+			RawPath: "https://github.com/slackapi/slack-cli/issues",
+		},
+		Info: func(ctx context.Context, clients *shared.ClientFactory) {
+			clients.IO.PrintInfo(ctx, false, fmt.Sprintf(
+				"%s\n%s\n",
+				style.Secondary("Ask questions, submit issues, or suggest features for the SLack CLI:"),
+				style.Secondary(style.Highlight("https://github.com/slackapi/slack-cli/issues")),
+			))
+		},
+		Trace: slacktrace.FeedbackMessage,
+		Ask: func(ctx context.Context, clients *shared.ClientFactory) (bool, error) {
+			clients.IO.PrintInfo(ctx, false, style.Sectionf(style.TextSection{
+				Emoji: "love_letter",
+				Text:  "We would love to know how things are going",
+				Secondary: []string{
+					"Share your experience with " + style.Commandf("feedback --name slack-cli-feedback", false),
+				},
+			}))
+			return false, nil
+		},
```

Comment:
note: Since we `Never` randomly ask about this survey, this function is never actually executed. But it's a required field.

Reply:
🤔 No blocker for this PR, but it might be interesting to replace the `Ask` parameter with a "string" for `style.TextSection` inputs instead.

IMO it's somewhat confusing to keep this prompt with the "Never" option otherwise 🔍

Reply:
This is a great suggestion. I agree, the `Ask` with `Never` is unnecessary. The `style.TextSection` also simplifies the implementation while keeping the prompt more consistent.

Honestly, I don't see us using the `Ask` in the originally intended way. If we don't leverage it in the future, we could also replace it with a single "ask" for feedback.